ush/python_utils/config_parser.py

```diff
@@ -214,10 +214,13 @@ def load_ini_config(config_file, return_string=0):
     """Load a config file with a format similar to Microsoft's INI files"""
 
     if not os.path.exists(config_file):
-        raise FileNotFoundError(dedent(f'''
-                                The specified configuration file does not exist:
-                                "{config_file}"'''
-        ))
+        raise FileNotFoundError(
+            dedent(
+                f'''
+                The specified configuration file does not exist:
+                "{config_file}"'''
```

Comment:
For consistency, would it be better to convert the single quotes to double quotes, then convert the double quotes around `{config_file}` to single quotes?

Reply:
Note that I did make these changes by hand, but run a python code formatter on all python files using`black --safe`. 
I do this from time to time but we should probably apply this on the client/server side and force a coding/formatting standard. Black has a preference for double quotes (single or triple) over single quotes which is why you are seeing those changes.

Reply:
Thanks for the information about the `black` python code formatter.  From [StackOverflow](https://stackoverflow.com/questions/71180810/python-black-code-formatter-is-there-any-way-to-apply-automatic-black-formatti), using Git hooks, it looks like it would be possible to run `black` on the python scripts before committing changes.  Is this python code formatter available on RDHPCS machines, or is this something that you run on your own machine before creating the PR?  If it is available on RDHPCS, then it should be fairly easy to set up a Git hook to run before committing (create a `.git/hooks/pre-commit` hook to use the formatter on the python files).

Do we want to use `black`, or is there another python code formatter that might be better?  Do we want to enforce what `black` recommends, or create our own standard?

I think this would be a good topic for the SRW meeting this week.